test/widgets/message_list_test.dart

```diff
@@ -1188,6 +1189,77 @@ void main() {
 
       debugNetworkImageHttpClientProvider = null;
     });
+
+
+    group('action sheet visual feedback', () {
+      late Message message;
+
+      setUp(() {
+        message = eg.streamMessage();
+      });
+
+      Color? getBackgroundColor(WidgetTester tester) {
+        final decoratedBox = tester.widget<DecoratedBox>(
+          find.descendant(
+            of: find.byType(MessageWithPossibleSender),
+            matching: find.byType(DecoratedBox),
+          ),
+        );
+        return (decoratedBox.decoration as BoxDecoration).color;
+      }
+
+      testWidgets('starts with transparent background', (tester) async {
+        await setupMessageListPage(tester, messages: [message]);
+
+        check(getBackgroundColor(tester),
+          because: 'Message should start with transparent background',
```

Comment:
The `because` isn't needed; the test description is sufficient. Let's remove the `because`. (Here and in the other added tests.)